plugins/guests/linux/cap/persist_mount_shared_folder.rb

```diff
@@ -8,14 +8,22 @@ module Cap
       class PersistMountSharedFolder
         extend SyncedFolder::UnixMountHelpers
 
+        @@logger = Log4r::Logger.new("vagrant::guest::linux::persist_mount_shared_folders")
+
         # Inserts fstab entry for a set of synced folders. Will fully replace
         # the currently managed group of Vagrant managed entries. Note, passing
         # empty list of folders will just remove entries
         #
         # @param [Machine] machine The machine to run the action on
         # @param [Map<String, Map>] A map of folders to add to fstab
         def self.persist_mount_shared_folder(machine, folders)
+          unless fstab_exists?(machine)
```

Comment:
Update the `unless` to an `if !fstab_exists?` simply due to the overall preference for `if` based conditionals